lib/recording/RecordingManager.js

```diff
@@ -19,6 +19,7 @@ const configRecording = config.get("recording");
 const kalturaTypes = require('../kaltura-client-lib/KalturaTypes');
 const unknownEntry = "UNKNOWN";
 const backendClient = require('../BackendClientFactory.js').getBackendClient();
+const maxAllowedMp4FilesQueueSize = 100;
```

Comment:
Should move to configure file in my opinion. not health to have hard coded numbers

Reply:
yes. done